ydb/docs/en/core/concepts/toc_i.yaml

```
@@ -19,7 +19,7 @@ items:
 - name: Change Data Capture (CDC)
   href: cdc.md
   when: feature_changefeed
-- name: Time to Live (TTL)
+- name: Time to Live (TTL) and Eviction to External Storage
```

```suggestion
- name: Time to live and eviction
```
That's too long for the sidebar.